targets/targets.json

```diff
@@ -7541,7 +7583,10 @@
                 "macro_name": "CLOCK_SOURCE"
             }
         },
-        "device_has_add": ["SERIAL_ASYNCH", "SERIAL_FC", "FLASH"],
+        "device_has_add": [
```

Comment:
Can this styling change be in separate commit please?

- device_has styling update
- serial_fc addition

Reply:
> Can this styling change be in separate commit please?

done